UI/CUIControls.cpp

```diff
@@ -1388,10 +1388,10 @@ namespace {
     GG::Y VERTICAL_SECTION_GAP(4);
 }
 
-ProductionInfoPanel::ProductionInfoPanel(const std::string& title, const std::string& point_units_str,
-                                         GG::X w, GG::Y h) :
-    CUIWnd(title, GG::X0, GG::Y0, GG::X(120), GG::Y(120),
-           GG::INTERACTIVE | GG::RESIZABLE | GG::DRAGABLE | GG::ONTOP | PINABLE),
+ProductionInfoPanel::ProductionInfoPanel(const std::string& title, const std::string& point_units_str, const GG::X x, const GG::Y y,
+    const GG::X w, const GG::Y h, const std::string& config_name) :  
```

Comment:
this line is parameters, not the initializer list, so fix the indenting to line up with the other parameters.